tools/sgcollect_info

```diff
@@ -149,6 +149,10 @@ def make_collect_gotool_pprof_tasks(zip_dir, sg_binary_path, sg_url):
         "raw",
     ]
 
+    #if sg_binary_path does not point to a file, generate a new path by starting with the location of sgcollect_info script
+    if not os.path.isfile(sg_binary_path):
+        sg_binary_path = os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "bin", os.path.basename(sg_binary_path))
```

Comment:
Can you split out `os.path.dirname(os.path.dirname(os.path.realpath(__file__)))` and `os.path.basename(sg_binary_path)` into a named variable?  Eg:

```
# the something, eg, /path/to/something
something = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))

# the filename of the sg_binary without any path information, eg, "sync_gateway"
sg_binary_filename = os.path.basename(sg_binary_path)

```